vundle.fish

```diff
@@ -0,0 +1,18 @@
+# SYNOPSIS
+#   Upon installation or loading, vundle checks for the existence
+#   of the Vundle.vim plugin and will notify if it is missing.
+#
+# USAGE
+#   $ vundle # runs PluginInstall to install missing plugins
```

Comment:
I have never seen an usage with a description, it seems like the explanation is part of the synopsis and not part of the usage.